bananas_cli/session.py

```diff
@@ -64,8 +65,11 @@ def tus_upload(self, upload_token, fullpath, filename):
                 metadata={"filename": filename, "upload-token": upload_token},
             )
             uploader.upload()
-        except TusCommunicationError:
-            log.exception(f"Failed to upload file '{filename}'")
+        except TusCommunicationError as e:
+            if self.verbose:
+                log.exception(f"Failed to upload file '{filename}'")
+            else:
+                log.error(f"Failed to upload file '{filename}': {e}")
```

Comment:
I am not sure this gives better errors that the user understands sufficiently. Do you have an example of this? In my experience, the `str(e)` often gives something impossible to understand :D

Reply:
My first iteration just said "failed to upload file {filename}", but that took away all agency from me as a command line user. I think if you are using the command line, you probably are more comfortable understanding error messages than a clicky person. 

Aanyway: 
If not specifying TUS url: 
`2021-04-12 17:21:01 ERROR    Failed to upload file 'xxxx.grf': Attempt to retrieve create file url with status 404`

With `--verbose` specified: 
```
2021-04-12 17:22:19 ERROR    Failed to upload file 'xxxx.grf'
Traceback (most recent call last):
  File "/code/bananas-frontend-cli/bananas_cli/session.py", line 62, in tus_upload
    uploader = tus.uploader(
  File "/code/bananas-frontend-cli/.env/lib/python3.8/site-packages/tusclient/client.py", line 52, in uploader
    return Uploader(*args, **kwargs)
  File "/code/bananas-frontend-cli/.env/lib/python3.8/site-packages/tusclient/uploader.py", line 124, in __init__
    self.url = url or self.get_url()
  File "/code/bananas-frontend-cli/.env/lib/python3.8/site-packages/tusclient/uploader.py", line 216, in get_url
    return self.create_url()
  File "/code/bananas-frontend-cli/.env/lib/python3.8/site-packages/tusclient/uploader.py", line 22, in _wrapper
    return func(*args, **kwargs)
  File "/code/bananas-frontend-cli/.env/lib/python3.8/site-packages/tusclient/uploader.py", line 232, in create_url
    raise TusCommunicationError(msg, resp.status_code, resp.content)
tusclient.exceptions.TusCommunicationError: Attempt to retrieve create file url with status 404
```

Typing https instead of http: 
`2021-04-12 17:23:49 ERROR    Failed to upload file 'v9point8.grf': HTTPSConnectionPool(host='172.17.0.2', port=1080): Max retries exceeded with url: /new-package/tus/ (Caused by SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1125)')) `

Wrong api url: 
`2021-04-12 17:28:29 ERROR    Cannot connect to host 172.17.0.2:443 ssl:default [Connect call failed ('172.17.0.2', 443)]`